<?xml version="1.0"?>
<launch>

    <arg name="namespace" default="bebop" />

    <remap from="camera" to="$(arg namespace)/image_raw"/>
    <remap from="actions" to="iisc_bebop_nav/cmd"/>
    <remap from="output_image" to="iisc_autonomous_nav/output_image"/>

    <node name="iisc_bebop_nav" pkg="iisc_autonav_outdoors" type="bebop_nav.py" output="screen">
        <remap from="cmd" to="/$(arg namespace)/cmd_vel"/>
        <remap from="land" to="$(arg namespace)/land"/>
        <remap from="takeoff" to="$(arg namespace)/takeoff"/>
        <remap from="cam_control" to="$(arg namespace)/camera_control"/>
        <remap from="battery" to="$(arg namespace)/states/common/CommonState/BatteryStateChanged"/>
        <rosparam command="load" file="$(find iisc_autonav_outdoors)/config/bebop_nav_vals.yaml" />
    </node>

    <node name="iisc_rf" pkg="iisc_autonav_outdoors" type="rf.py" output="screen">
        <remap from="logits" to="/iisc_rf/DNN_logits"/>
        <rosparam command="load" file="$(find iisc_autonav_outdoors)/config/rf_settings.yaml" />
    </node>

    <rosparam command="load" file="$(find iisc_autonav_outdoors)/config/iisc_autonomous_nav.yaml" />

</launch>
